aws/resource_aws_cognito_user_pool.go

```diff
@@ -131,6 +131,7 @@ func resourceAwsCognitoUserPool() *schema.Resource {
 				Type:     schema.TypeList,
 				Optional: true,
 				MaxItems: 1,
+				Computed: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"reply_to_email_address": {
```

Comment:
In order to get our testing working I added `""` as a valid value in the validation function.